engine/graphics/src/null/graphics_null.cpp

```diff
@@ -852,10 +852,12 @@ namespace dmGraphics
         {
             case ShaderDesc::SHADER_TYPE_VERTEX:
                 return language == ShaderDesc::LANGUAGE_GLSL_SM330 ||
-                       language == ShaderDesc::LANGUAGE_GLES_SM300;
+                       language == ShaderDesc::LANGUAGE_GLES_SM300 ||
+                       language == ShaderDesc::LANGUAGE_SPIRV;
```

Comment:
TODO (future): The null adapter should support all languages and a mechanism to turn on/off specific languages, so that we can test more things. The actual shader code is never run anyway.